v2/emailpassword/appinfo.mdx

```diff
@@ -78,6 +78,10 @@ If you want to change the `/auth` to something else, then you must set this valu
 Remember to set the same value for this param on the backend and the frontend.
 :::
 
+:::caution
```

Comment:
- So? Why is this a caution - what makes this risky? You need to explain. 
- Also, in your example, don't use `SOME_PATH`. Use an actual string. Explain that if first it is "/auth", then you change it to "/supertokens" (for example), then your refresh endpoint will be updated from "/auth/session/refresh" to "/supertokens/session/refresh". This then implies that your previously issued refresh tokens will no longer get sent to the new API endpoint because they are associated with "/auth/session/refresh" - causing a user logout.